index/scorch/merge.go

```diff
@@ -429,6 +430,7 @@ type segmentMerge struct {
 	oldNewDocNums map[uint64][]uint64
 	new           segment.Segment
 	notifyCh      chan *mergeTaskIntroStatus
+	mmaped        bool
```

Comment:
I'd rather we keep the type of this variable as uint for consistency.

Reply:
Done.